pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2060,6 +2060,10 @@ type KerberosSpec struct {
 	// +kubebuilder:default="nfs"
 	PrincipalName string `json:"principalName"`
 
+	// Domain is used to setup idmapd.conf so that the idmap can map Kerberos principals to uid/gids.
```

Comment:
IMO, not necessary to provide details about the underlying implementation here, which is what this seems like to me. Correct me if I'm wrong though. What's most important for users is to know how to fill in the value.

```suggestion
	// DomainName should be set to the Kerberos FQDN.
```

Reply:
Will do.